BUILDER_DEV.md

```diff
@@ -1,390 +1,294 @@
+* From a terminal, run
 
 ```
-export HAB_AUTH_TOKEN=<your github token>
-export HAB_BLDR_URL=http://localhost:9636
-export HAB_ORIGIN=<your origin>
+$ hab origin key import
 ```
 
-Now run `sudo -E make bldr-run` from the root of your hab repo.
+* Paste the production core origin public key contents
+* Hit Ctrl + D twice
+* Run this again:
 
-The first time this command runs, it will create the required databases. Let it run for a while, and then re-start it if there are errors (this is normal for the first time setup).
+```
+$ hab origin key import
+```
 
-When `make bldr-run` can proceed with all the services up and running, you are ready to proceed to the next step.
+* Paste the production core origin private key contents
+* Hit Ctrl + D twice
+* Run
+```
+$ hab install core/hab-backline
+$ cd /hab/cache/artifacts
+$ hab pkg upload -c stable core-hab-backline...hart
+```
+* You will need to follow this same process for any dependencies of anything you want to build locally - the sample app I will use below depends on [core/scaffolding-node](https://bldr.habitat.sh/#/pkgs/core/scaffolding-node/latest), [core/git](https://bldr.habitat.sh/#/pkgs/core/git/latest), [core/node](https://bldr.habitat.sh/#/pkgs/core/node/latest), [core/busybox-static](https://bldr.habitat.sh/#/pkgs/core/busybox-static/latest) - so I will follow the same procedure to download them from production builder, then upload them to my local builder.
 
-## Create your origin
-There are a couple of ways to do this.
+### Create a Project
+* In UI
+* Click on the Origin your created in "Creating an Origin"
+* Click on "Connect a plan file"
+* Add in a github repo with a plan.sh file in it (I use "nellshamrell/sample-node-app")
+* Click "Save Connection" - this will bring you to the view for that origin
+* Click on "Latest"
+* Click on "Build latest version"
+* Click on "Build Jobs" and "View the output" to see the job in progress
+* The job should complete successfully! Congrats, you have a working build!
 
-### From the command line
-1. Open a new terminal window.
-1. Export the `HAB_ORIGIN` and `HAB_AUTH_TOKEN` as above
-1. Create an origin in the DB by issuing the following command:
+## Unsupported Dev Environments
 
-    ```
-    http POST http://localhost:9636/v1/depot/origins Content-Type:application/json Authorization:Bearer:${HAB_AUTH_TOKEN} name=${HAB_ORIGIN}
-    ```
+Maintainers have historically been able to use alternative development environment setups. If you would like to explore using other OS's or Docker or Vagrant - please check out these links. **Do remember that these are NOT officially supported by the Habitat maintainers - use at your own risk!**
 
-    The response should be something like:
+* [Other Operating Systems](BUILDING.md)
+* [Container Dev Environment](BUILDER_CONTAINER.md)
 
-    ```
-    HTTP/1.1 201 Created
+## General build notes
```

Comment:
Is is worth adding anything about the ports that need to be forwarded and how to achieve that to access the web UI from outside the VM?

Reply:
We could, but I'd rather they be added after this pull request is resolved/merged. Good to keep in mind!